File: app/src/main/java/ua/com/amicablesoft/android/wr/ui/AddCompetitionPresenter.java
package ua.com.amicablesoft.android.wr.ui;

import ua.com.amicablesoft.android.wr.dal.Repository;

/**
 * Created by olha on 2/28/17.
 */

public class AddCompetitionPresenter {

    public boolean isCompetitionValid(String competition) {
        return competition.length() > 1;
    }

    public void callWriteNewCompetition(String competition) {
        Repository repository = new Repository();
        repository.writeNewCompetition(competition);

    }
}

File: app/src/main/java/ua/com/amicablesoft/android/wr/ui/AddCompetitionPresenter.kt
/*
 * Created by Olha Tymoshenko on 5/25/18 3:40 PM.
 * Copyright (c) 2018. All rights reserved.
 */

package ua.com.amicablesoft.android.wr.ui

import ua.com.amicablesoft.android.wr.dal.Repository

class AddCompetitionPresenter {

    fun isCompetitionValid(competition: String): Boolean {
        return competition.length > 1
    }

    fun callWriteNewCompetition(competition: String) {
        val repository = Repository()
        repository.writeNewCompetition(competition)

    }
}